src/libstore/globals.hh

```diff
@@ -159,6 +174,18 @@ public:
         )",
         {"build-max-jobs"}};
 
+    MaxSubstitutionJobsSetting maxSubstitutionJobs{
```

Comment:
```suggestion
    MaxBuildJobsSetting maxSubstitutionJobs{
```
and remove `MaxBuildJobsSetting` since it appears to be exactly the same as `MaxSubstitutionJobsSetting`.

Reply:
Considering other comments, I removed the `MaxSubstitutionJobsSetting` and used `Setting<unsigned int>` instead.